backend/stakepoold/rpc/rpcserver/context.go

```diff
@@ -397,6 +404,49 @@ func (ctx *AppContext) GetTickets(includeImmature bool) ([]*chainhash.Hash, erro
 	return tickets, nil
 }
 
+func (ctx *AppContext) GetTicketInfo(ticketHash string) (ticketInfo *TicketInfo, err error) {
+	hash, err := chainhash.NewHashFromStr(ticketHash)
+	if err != nil {
+		log.Errorf("GetTicketInfo: Failed to parse ticket hash: %v", err)
+		return
+	}
+
+	res, err := ctx.WalletConnection.RPCClient().GetTransaction(hash)
+	if err != nil {
+		log.Errorf("GetTicketInfo: GetTransaction rpc failed: %v", err)
+		return
+	}
+
+	// get txout addresses using tx hex
+	msgTx, err := MsgTxFromHex(res.Hex)
+	if err != nil {
+		return nil, fmt.Errorf("GetTicketInfo: MsgTxFromHex failed for %v: %v", res.Hex, err)
```

Comment:
Until here you were logging the error before returning it.  I think that's the right thing to do here, as I don't think it is logged anywhere else inside stakepoold. So please continue logging.